discovery-service/src/main/java/org/zowe/apiml/discovery/eureka/RefreshablePeerEurekaNodes.java

```diff
@@ -45,19 +46,12 @@ public RefreshablePeerEurekaNodes(final PeerAwareInstanceRegistry registry,
 
     @Override
     public PeerEurekaNode createPeerEurekaNode(String peerEurekaNodeUrl) {
-        JerseyReplicationClient replicationClient = JerseyReplicationClient
-            .createReplicationClient(serverConfig, serverCodecs,
-                peerEurekaNodeUrl);
-
-        this.replicationClientAdditionalFilters.getFilters()
-            .forEach(replicationClient::addReplicationClientFilter);
-
+        HttpReplicationClient replicationClient = Jersey3ReplicationClient.createReplicationClient(serverConfig, serverCodecs, peerEurekaNodeUrl);
```

Comment:
I see that `replicationClientAdditionalFilters` is not being used anymore, is this intentional?